extensions/observability-devservices/deployment/src/main/java/io/quarkus/observability/deployment/ObservabilityDevServiceProcessor.java

```diff
@@ -250,14 +257,24 @@ public DevServicesResultBuildItem.RunningDevService get() {
                 .locateContainer(
                         capturedDevServicesConfiguration.serviceName(), capturedDevServicesConfiguration.shared(),
                         LaunchMode.current(), (p, ca) -> config.putAll(dev.config(p, ca.getHost(), ca.getPort())))
-                .map(new Function<String, DevServicesResultBuildItem.RunningDevService>() {
-                    @Override
-                    public DevServicesResultBuildItem.RunningDevService apply(String cid) {
-                        log.infof("Dev Service %s re-used, config: %s", devId, config);
-                        return new DevServicesResultBuildItem.RunningDevService(Feature.OBSERVABILITY.getName(), cid,
-                                null, config);
-                    }
+                .map(cid -> {
+                    log.infof("Dev Service %s re-used, config: %s", devId, config);
+                    return new DevServicesResultBuildItem.RunningDevService(Feature.OBSERVABILITY.getName(), cid,
+                            null, config);
                 })
+                .or(() -> ComposeLocator.locateContainer(composeProjectBuildItem,
+                        List.of(capturedDevServicesConfiguration.imageName(), "lgtm"), LaunchMode.current())
```

Comment:
It shouldn't be "lgtm", as this processor is meant to handle any kind of DevResource.
If `ContainerConfig::label` works for you, use that, or add a new method. Or perhaps "devId" ...

Reply:
We can keep only the `capturedDevServicesConfiguration.imageName()`?